tutorials/abandoned-browse-ccdp/braze-campaign.md

```diff
@@ -5,89 +5,92 @@ title: Braze Campaign Setup
 
 This guide will walk you through setting up and testing an abandoned browse campaign in Braze using the product view data synced from Census.
 
-## Setting Up the Campaign
+## Setting up the campaign
 
 1. Log into your Braze dashboard
-2. Navigate to **Campaigns** and click "Create Campaign"
-3. Select "Email Campaign" as the channel
-4. Name your campaign (e.g., "Abandoned Browse - Product Reminder")
+2. Navigate to **campaigns** and click **Create campaign**
+3. Select **Email** as the channel
+4. Name your campaign (e.g. "Abandoned browse - product reminder")
 
-## Configure Campaign Trigger
+![Create Campaign](images/retl-braze-create-campaign.png)
 
-1. In the "Delivery" section, select "Action-Based"
+## Configure campaign trigger
+
+1. In the **Delivery** section, select **Action-Based**
 2. Set up the trigger criteria
 
-## Create Email Template
+## Create email template
 
-![Braze Campaign Builder](images/retl-braze.png)
+![Braze Campaign Builder](images/retl-braze-preview.png)
 
-1. Click "Edit Campaign" in the campaign builder and click "Edit Email Body"
-2. Design your email using Braze's visual editor or use the following HTML:
+1. Click **Edit Campaign** in the campaign builder and click **Edit Email Body**
+2. Design your email using Braze's visual editor:
    ```html
    Subject: Don't miss out on {{custom_attribute.${product}}}!
    
-   Hello!
-   
-   We noticed you were checking out {{custom_attribute.${product}}}. 
+   Hey {{${first_name}}} we saw you were interested in {{custom_attribute.${PRODUCT}}} 
    
    <a href="{{custom_attribute.${product_url}}}?abandonedEmail=true">
-     Take another look
+     You spent {{custom_attribute.${TIME_ENGAGED_IN_S}}} seconds checking it out.... Why not check it out once more before it sells out?!!
    </a>
-   ```
+   ``
 
 3. Add personalization:
-   - Product name using `{{custom_attribute.${product}}}`
-   - Product URL using `{{custom_attribute.${product_url}}}`
-   - Add dynamic product images if available
+   - product name using `{{custom_attribute.${product}}}`
```

Comment:
sorry, I was unclear, these ones should have stayed capitalised 😅 they're fine like this now, I might update them myself later